bindings/python/spatial/expose-explog.cpp

```diff
@@ -25,7 +25,7 @@ namespace pinocchio
               "Jacobian of exp(R) which maps from the tangent of SO(3) at exp(v) to"
               " the tangent of SO(3) at Identity.");
       
-      bp::def("log3",(Eigen::Vector3d (*)(const Eigen::MatrixBase<Eigen::Matrix3d> &))&log3<Eigen::Matrix3d>,
+      bp::def("log3",&log3_proxy,
```

Comment:
Why? I prefer to use cast here because I will then expose new signature for types other than double.

Reply:
It was crashing, I could not understand why. This was the quickest way to fix it. But I can come back on this later